src/server/graphql/resolvers/__tests__/resolveStartOfWeek.test.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-env mocha */
+/* global expect, testContext */
+/* eslint-disable prefer-arrow-callback */
+
+import moment from 'moment'
+
+import {resolveStartOfWeek} from '../index'
+
+describe(testContext(__filename), function () {
+  before(function () {
+    this.emptyCycle = {}
+    this.cycle = {}
+    this.previousCycle = {}
+    this.nextCycle = {}
+    this.cycle.startTimestamp = '2017-07-27' // Thursday
+    this.nextCycle.startTimestamp = '2017-07-28' // Friday
+    this.previousCycle.weekStartedAt = '2017-07-17'
```

Comment:
Also note that while this might work for the tests, it's not testing quite the same behavior as the real system has -- `startTimestamp` values are dates rather than strings.